datadog_checks_dev/datadog_checks/dev/tooling/e2e/agent.py

```diff
@@ -14,25 +32,55 @@ def get_rate_flag(agent_version):
         return 'check_rate'
 
 
-def get_agent_exe(agent_version, platform='linux'):
-    if platform == 'windows':
-        return ''
-    elif platform == 'mac':
-        return ''
+def get_agent_exe(agent_version, platform=LINUX):
+    if platform == WINDOWS:
+        if agent_version >= 6:
+            return r"C:\Program Files\Datadog\Datadog Agent\embedded\agent.exe"
+        else:
+            # [TODO] Actually get the path here
+            pass
+    elif platform == MAC:
+        if agent_version >= 6:
+            return 'datadog-agent'
+        else:
+            return 'dd-agent'
     else:
         if agent_version >= 6:
             return '/opt/datadog-agent/bin/agent/agent'
         else:
             return '/opt/datadog-agent/agent/agent.py'
 
 
-def get_agent_conf_dir(check, agent_version, platform='linux'):
-    if platform == 'windows':
-        return ''
-    elif platform == 'mac':
-        return ''
+def get_agent_conf_dir(check, agent_version, platform=LINUX):
+    if platform == WINDOWS:
+        if agent_version >= 6:
+            return r'C:\ProgramData\Datadog\conf.d\{}.d'.format(check)
+        else:
+            return r'C:\ProgramData\Datadog\conf.d'
+    elif platform == MAC:
+        if agent_version >= 6:
+            return '/opt/datadog-agent/etc/conf.d/{}.d'.format(check)
+        else:
+            return '/opt/datadog-agent/etc/conf.d'
```

Comment:
Are you sure it's the same as A6?

Reply:
Indeed, according to our docs page - https://docs.datadoghq.com/agent/basic_agent_usage/osx/?tab=agentv5#configuration